koku/sources/api/view.py

```diff
@@ -121,6 +122,11 @@ class SourcesViewSet(*MIXIN_LIST):
     filterset_class = SourceFilter
     http_method_names = HTTP_METHOD_LIST
 
+    @action(methods=["get"], detail=False, permission_classes=[AllowAny], url_path="aws-s3-regions")
+    def aws_s3_regions(self, request):
+        regions = get_available_regions("s3")
+        return Response({"regions": regions})
```

Comment:
```suggestion
        return ListPaginator([{"regions": regions}], request).paginated_response
```
For completeness sake, we'll want to use our Paginators here.

Which comes from:
```
from api.common.pagination import ListPaginator
```

The other methods on this endpoint do not use this Paginator, but they _should_. This was a miss back in the day which we never corrected.